client/src/pages/ContactUs.tsx

```diff
@@ -0,0 +1,37 @@
+import "../pages/ContactUs.css";
+function ContactUs() {
+  return (
+    <div className="div-cu">
+      <section className="section-1">
+        <div className="left-part">
+          <input className="name-cu" type="text" placeholder="Nom" />
+          <input className="first-name-cu" type="text" placeholder="Prenom" />
+          <input className="mail-cu" type="text" placeholder="Adresse mail" />
+        </div>
+        <div className="right-part">
+          <p>Des series que tu souhaite voire sur le site ?</p>
+          <input type="text" placeholder="Nom d'une série" />
+          <input type="text" placeholder="" />
```

Comment:
If there is not need for a placeholder, don't add the attribute `placeholder=""`